test/factories/iterations.rb

```diff
@@ -0,0 +1,18 @@
+FactoryBot.define do
+  factory :iteration do
+    iteration_time ""
+    iteration_cost 1.5
```

Comment:
The factory is invalid

Reply:
Factories should be able to make the objects they are supposed to make, please ensure this factory can make Iterations correctly.